slack/tool-bot.gpt

```diff
@@ -0,0 +1,44 @@
+---
+Name: Slack Bot
+Description: Tools for interacting with Slack as an App
+Metadata: bundle: true
+Share Tools: Send Direct Message As Bot
```

Comment:
```suggestion
Share Tools: Send Direct Message As Bot, List Users As Bot, Search Users As Bot
```

Reply:
Intentionally avoid adding these two tools into share tools so that these are not indexed in obot. For now I only want to expose `Send Direct Message As Bot` since that's the only use case we have as a bot.

Reply:
Oh sorry, I missed the comment you made about this in the PR description

Reply:
No problem 👍